charts/spire/README.md

```diff
@@ -292,6 +292,10 @@ Kubernetes: `>=1.21.0-0`
 | spire-server.controllerManager.image.repository | string | `"spiffe/spire-controller-manager"` | The repository within the registry |
 | spire-server.controllerManager.image.tag | string | `"0.2.2"` | Overrides the image tag |
 | spire-server.controllerManager.image.version | string | `""` | This value is deprecated in favor of tag. (Will be removed in a future release) |
+| spire-server.controllerManager.livenessProbe.httpGet.path | string | `"/healthz"` |  |
```

Comment:
This whole pr looks very nice. Thanks. :)

The one thing I think we may want to do/not do is to make the paths/ports/scheme overridable. there isn't a different setting the user would ever need to set for those? so maybe we hard code those still.

Reply:
And maybe make the rest of the missing liveness/readyness/startup probe settings configurable?

Reply:
So @kfox1111 , are you suggesting format something like this?:

```
          readinessProbe:
            httpGet:
              scheme: HTTP
              path: /ready
              port: healthz
            failureThreshold: {{ .Values.startupProbe.failureThreshold }}
            initialDelaySeconds: {{ .Values.startupProbe.initialDelaySeconds }}
            periodSeconds: {{ .Values.startupProbe.periodSeconds }}
            successThreshold: {{ .Values.startupProbe.successThreshold }}
            timeoutSeconds: {{ .Values.startupProbe.timeoutSeconds }}
 ```

Reply:
yeah. that kind of thing.

Reply:
That way, the user can configure the settings that may differ cluster to cluster (slower cluster, higher timeouts) but not have to worry about the stuff that is internal to the pod that won't change on them.